domain/cloud/state/state.go

```diff
@@ -500,6 +500,24 @@ func (st *State) UpsertCloud(ctx context.Context, cloud cloud.Cloud) error {
 	return errors.Trace(err)
 }
 
+// CreateCloud saves the specified cloud.
+// Exported for use in the related cloud bootstrap package.
+func CreateCloud(ctx context.Context, tx *sql.Tx, cloudUUID string, cloud cloud.Cloud) error {
+	if err := upsertCloud(ctx, tx, cloudUUID, cloud); err != nil {
```

Comment:
My preference here would be fmt.Errorf. Annotate needs to die as it looses information.  We should also include more context in the error strings. Specifically the cloud uuid.

Reply:
Added cloud uuid in message. As discussed, will leave Annotate for now, till we agree on a consistent approach across the code base.